packages/core/src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -217,6 +220,8 @@ export interface SamCliLocalInvokeInvocationArguments {
     name?: string
     /** AWS region */
     region?: string
+    /** Runtime to use for testing. Can be different from the one specified in the template. */
+    runtime?: Runtime
```

Comment:
What is meant by "for testing"? Isn't this a feature available to users (not just for tests)?

Reply:
Makes more sense. Thanks. Updated.